src/bvals/comms/build_boundary_buffers.cpp

```diff
@@ -44,70 +44,69 @@ template <BoundaryType BTYPE>
 void BuildBoundaryBufferSubset(std::shared_ptr<MeshData<Real>> &md,
                                Mesh::comm_buf_map_t &buf_map) {
   Mesh *pmesh = md->GetMeshPointer();
-  ForEachBoundary<BTYPE>(
-      md, [&](sp_mb_t pmb, sp_mbd_t /*rc*/, nb_t &nb, const sp_cv_t v) {
-        // Calculate the required size of the buffer for this boundary
-        int buf_size = GetBufferSize(pmb, nb, v);
-
-        // Add a buffer pool if one does not exist for this size
-        if (pmesh->pool_map.count(buf_size) == 0) {
-          pmesh->pool_map.emplace(std::make_pair(
-              buf_size, buf_pool_t<Real>([buf_size](buf_pool_t<Real> *pool) {
-                using buf_t = buf_pool_t<Real>::base_t;
-                // TODO(LFR): Make nbuf a user settable parameter
-                const int nbuf = 200;
-                buf_t chunk("pool buffer", buf_size * nbuf);
-                for (int i = 1; i < nbuf; ++i) {
-                  pool->AddFreeObjectToPool(
-                      buf_t(chunk, std::make_pair(i * buf_size, (i + 1) * buf_size)));
-                }
-                return buf_t(chunk, std::make_pair(0, buf_size));
-              })));
-        }
-
-        const int receiver_rank = nb.snb.rank;
-        const int sender_rank = Globals::my_rank;
-
-        int tag = 0;
+  ForEachBoundary<BTYPE>(md, [&](auto pmb, sp_mbd_t /*rc*/, nb_t &nb, const sp_cv_t v) {
```

Comment:
I didn't carefully check, but it looks like aside from switching to pointers all the changes in this file are just whitespace related?

Reply:
Yeah git and clang-format somehow interacted to cause an ugly looking diff. This happened when I merged your multigrid branch in. Not sure why. But the only thing I changed in, e.g., the `ForEachBoundary` call was replace `std::shared_ptr<MeshBlock>` with `auto` in the lambda.